docs/articles/documentation/using-testcafe/common-concepts/reporters.md

```diff
@@ -44,5 +45,30 @@ You can install reporter packages from npm as you would install any other plugin
 
 When running tests, you can select a reporter to generate test reports.
 You can do this by using the
-[reporter](../command-line-interface.md#-r-namefile---reporter-namefile) command line option or the
-[runner.reporter](../programming-interface/runner.md#reporter) API method.
\ No newline at end of file
+[-r (--reporter)](../command-line-interface.md#-r-namefile---reporter-namefile) command line option or the
+[runner.reporter](../programming-interface/runner.md#reporter) API method.
+
+## Report Content
+
+> The description below applies to the most popular reporters. Specific reporters may produce output that differs from the one described here.
+
+A typical reporter outputs information about the browsers where test were run followed by the list of passed and failed tests.
```

Comment:
1) you I describe only reports for failed tests in this section so may be we need to give it more specific title
2) IMO this info is  quite general: reports `may also include` or may not, `most popular`.. and so on.
3) Maybe we should  add report examples for cases described in [`Reports may also include`](https://github.com/DevExpress/testcafe/pull/2932/files#diff-ca20d2ebd78df1d64222528b9352ecc8R69) section (warnings, links to screenshots, quarantine mode)